tests/test_bgp_af.rb

```diff
@@ -228,6 +308,39 @@ def test_next_hop_route_map
                    'configured and should not be')
   end
 
+  def test_next_hop_route_policy
```

Comment:
Are next-hop route-map and next-hop route-policy two different platforms' way of configuring the same behavior? If so I'd like to see a single test case for both rather than mutually exclusive test cases.